### PR TITLE
Move dist version to separate file and share it with Android SDK

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

**Review by Vezzra — APPROVED** [review]
Looks reasonable. Works fine on macOS (can build SDK, and build  and run FO with that SDK).

Aside from a typo in a comment, LGTM.

File: Version.cmake [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This file contains CMake variable defenition of SDK version
```
[the remark this answers is not in the document]
Comment (geoffthemedio) [reply Version.cmake:1]: @Vezzra you can do a suggestion with the +/- button eg.
```suggestion
# This file contains CMake variable definition of SDK version
```

[the remark this answers is not in the document]
Comment (Vezzra) [reply Version.cmake:1]: Ah, ok, thanks! 👍

[the remark this answers is not in the document]
Comment (o01eg) [reply Version.cmake]: Fixed.

**Comment by geoffthemedio** [comment]
no other comment

**Pull request merged**